languages-and-frameworks/dockerfile.html.md.erb

```diff
@@ -31,7 +31,7 @@ Wrote config file fly.toml
 ? Would you like to deploy now?
 ```
 
-Let `fly launch` generate an app name for you or pick your own.
+Let `fly launch` generate an app name for you or pick your own. You may need to explicitily specify Dockerfile path using `--dockerile` flag if fly doesn't detect Dockerfile automatically.
```

Comment:
The issue isn't that fly doesn't detect the Dockerfile automatically, successfully launching a Rails application typically requires the setting of at least one secret (RAILS_MASTER_KEY), and either the deployment of a database or the creation of a volume.  That's why the scanner for Rails is run before the Dockerfile scanner.